docs/getting-started/kubernetes.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Kubernetes
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+import K8sIngressInstallExample from "/examples/k8s/install-ingress.mdx";
+import K8sGatewayAPIInstallExample from "/examples/k8s/install-gateway-api.mdx";
+
+import K8sIngressModifyHeadersExample from "/examples/k8s/http-request-headers.mdx";
+import K8sGatewayModifyHeadersExample from "/examples/k8s/http-request-headers-gateway.mdx";
+
+import K8sIngressRedirectExample from "/examples/k8s/http-request-redirect.mdx";
+import K8sGatewayRedirectExample from "/examples/k8s/http-request-redirect-gateway.mdx";
+
+import StaticDomainKubernetesExample from "/examples/k8s/http-static-domain.mdx";
+import StaticDomainKubernetesGatewayExample from "/examples/k8s/http-static-domain-gateway.mdx";
+
+import StaticDomainKubernetesPathExample from "/examples/k8s/http-static-domain-path.mdx";
+import StaticDomainKubernetesGatewayPathExample from "/examples/k8s/http-static-domain-gateway-path.mdx";
+
+# Quickstart: Kubernetes Operator
+
+This quickstart will use the [ngrok Kubernetes Operator](https://github.com/ngrok/kubernetes-ingress-controller) to put an application running on a Kubernetes Cluster on the internet and add some basic policies
+
+## What is the ngrok Kubernetes Operator?
+
+The ngrok Kubernetes Operator is the best way to use ngrok if your applications run as Services in a Kubernetes cluster. It supports both [Ingress CRDs](https://kubernetes.io/docs/concepts/services-networking/ingress/) as well as the new [Gateway API](https://gateway-api.sigs.k8s.io/). The ngrok Kubernetes Operator is deployed to a cluster and allows developers to self-service public endpoints to their apps and services using a shared ngrok account. The operator watches for Kubernetes objects and creates the corresponding ngrok endpoints and configurations.
+
+## Step 1: Install Kubernetes Operator
+
+Installing the ngrok Operator is easy using Helm.
+
+<Tabs groupId="k8s" queryString="k8s-install">
+	<TabItem value="ingress" label="Ingress Controller" default>
+		<K8sIngressInstallExample />
+	</TabItem>
+	<TabItem value="gatewayAPI" label="Gateway API">
+		<K8sGatewayAPIInstallExample />
+	</TabItem>
+</Tabs>
+
+<br />
+
+## Step 2: Connect your service to an Edge
+
+This would create an Edge on with the Domain `example.ngrok.app` that routes traffic to the Kubernetes service `example-service` running on port 80.
+
+<Tabs groupId="k8s" queryString="k8s-basic">
+	<TabItem value="ingress" label="Ingress Controller" default>
+		<StaticDomainKubernetesExample />
+	</TabItem>
+	<TabItem value="gatewayAPI" label="Gateway API">
+		<StaticDomainKubernetesGatewayExample />
+	</TabItem>
+</Tabs>
+
+## Step 3: Route Traffic based on a path
+
+This would create an Edge on with the Domain `example.ngrok.app` that routes traffic from example.ngrok.app/foo to the Kubernetes service `example-service` running on port 80.
```

Comment:
```suggestion
This would create an Edge on with the Domain `example.ngrok.app` that routes traffic from `example.ngrok.app/foo` to the Kubernetes service `example-service` running on port `80`.
```